src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -726,7 +726,7 @@ int SolverInterfaceImpl::setMeshVertex(
   int           index   = -1;
   MeshContext & context = _accessor->usedMeshContext(meshID);
   mesh::PtrMesh mesh(context.mesh);
-  PRECICE_DEBUG("MeshRequirement: {}", context.meshRequirement);
+  PRECICE_DEBUG("MeshRequirement: {}", fmt::underlying(context.meshRequirement));
```

Comment:
This should use the ostream formatter

```suggestion
  PRECICE_DEBUG("MeshRequirement: {}", fmt::streamed(context.meshRequirement));
```

Or we need to add this new template specialization below:

https://github.com/precice/precice/blob/0590b349df3e28e0a9a99e4e5e1a05256565d494/src/mapping/Mapping.hpp#L191